app/main.py

```diff
@@ -16,4 +16,22 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+mutable_vars = []
+immutable_vars = []
+
+variables_to_check = [lucky_number, pi, one_is_a_prime_number, name,
+                      my_favourite_films, profile_info,
+                      marks, collection_of_coins]
+
+for variable_to_check in variables_to_check :
+    if isinstance(variable_to_check, (list, dict, set)):
+        mutable_vars.append(variable_to_check)
+    else:
+        immutable_vars.append(variable_to_check)
+
+sorted_variables = {
```

Comment:
just populate the dictionary without any additional logic

Reply:
@Polyakiv-Andrey 
Got it.
I reduced the source code to a minimum.

Reply:
@Polyakiv-Andrey Could you please take a look at the fixed version of the code? I will be very grateful!